src/pages/JoseConnect.vue

```diff
@@ -22,69 +38,79 @@
 <script>
 export default {
   metaInfo: {
-    title: 'Demo Connect'
+    title: 'Jurra Connect'
   },
   data: function () {
     return {
       numberRepos: 3
     }
+  },
+  mounted () {
+    // console.log ('results: ' + JSON.stringify(this.$page))
+    console.log ('results: ' +
+      JSON.stringify(this.$page.gitapi.repos.repositories.nodes[4].docs.folders[0].lang))
+    console.log ('results: ' +
+      JSON.stringify(this.$page.gitapi.repos.repositories.nodes[4].docs.folders[0].folder.files[0]))
   }
 }
```

Comment:
I cloned your fork, tested this and it throws the following error: 
```
vue.runtime.esm.js?2b0e:619 [Vue warn]: Error in mounted hook: "TypeError: Cannot read property 'repositories' of undefined"
```

Reply:
Yes - I passed a tiny correction that fixed this oversight, but found you'd moved on for this branch, I thinking putting in that cards view you showed. 

Doesn't merge that easily, so I'll test it and bring it over, close the pull request...